src/vivarium_census_prl_synth_pop/components/person.py

```diff
@@ -88,30 +82,23 @@ def on_time_step(self, event: Event) -> None:
         """
 
         persons = self.population_view.get(event.index)
```

Comment:
Can you change this to just pop or population?  The naming is old and that keeps it clear and consistent across components.  I'm updating this in the other components.